frontends/web/src/routes/account/send/services.ts

```diff
@@ -18,6 +18,11 @@
 import { alertUser } from '../../../components/alert/Alert';
 import { i18n } from '../../../i18n/i18n';
 
+export type TSignProgress = {
+  steps: number;
+  step: number;
+}
```

Comment:
this type is coming from the backend api and should be in api/devicessync.ts

Reply:
Makes sense 👍